### DISCUSSION TITLE
Flashing Newt Display

### DISCUSSION THREAD

**Discussion by TalonCard** [Phambili-Tech/Newt_Display#52] [Q&A]
I grabbed 2x Newts recently, one has some very odd behaviour while the other operates fine.

What it does is that if a button is pressed, the screen will start alternating between full-black pixels and then half black pixels. Alternating once per sec. It'll keep doing this for 20 sec. It'll do the same flashing while booting or performing updates.

Another clue, the interface seems to lag when it's doing this alternating flashing. It becomes unresponsive and seems to take double taps from the sensors when navigating the menus.

I'm only a very amateur coder or electronics guy, but I figure this has something to do with the 1hz RTC and whether the CPU of the ESP32 is doing any crunching, and likely centred around hardware as the other one running the same software all checks out.
 
I couldn't see any dry solder joints, and the other Newt running the same software doesn't have the same issue so I'm at a bit of a loss as to where to start. Happy to get my iron out if a dry solder joint is the most likely culprit, but have never mucked with much surface mount stuff before. When comparing the two, I can't see anything obviously out of place, but I haven't run my multimeter over it to compare.

I have gone through the settings, just in case there was some dimming option I missed. I did try the dark mode (Which looks pretty great I think) but it operated the same, just with inverted pixels

Tried factory reset, force downloading the latest firmware, lettting the device discharge overnight, I also tried the RC and then upgrading from there. It operates the same whether I'm powering the unit via Type C or a battery pack

Initially I thought this was something to do with not having a battery in the unit, or some sort of way to indicate that you've pushed a button recently and didn't try the other one I had.

Video attached if it'll help, but I'm all open to ideas and really hoping I haven't missed something obvious

(Also, a magnetic 3d printed case to accomodate those 4000mah batteries is coming, just need to iron out a few more kinks)
[20221018 201833.webm](https://user-images.githubusercontent.com/116068528/196392036-f70045d9-662e-4c6c-89ed-c9cf042933b5.webm)

**Comment by darianbjohnson** [comment]
That's weird. We had another user with a similar issue; we replaced his unit and he's shipping the defective unit to me for analysis and repair.  Can you provide your contact information to me via the Contact Form on the crowdsupply website (https://www.crowdsupply.com/phambili/newt/ask-question)? I think I'd like to send a replacement screen to you, to see if that fixes the issue... and if not, then maybe we'll switch out the whole unit.

**Comment by darianbjohnson** [comment]
Also - those 4000 mAH batteries look awesome - and look like a great size for the Newt... can you share where you get them? I'm sure others will be interested.

**Reply by TalonCard** [reply]
Thank you kindly! I'll jump on that form shortly.

As for the battery, this is the part I had ordered. It was just luck that the dimensions match the Newt so perfectly!

https://ecocell.com.au/product/lipo-4000-606090/ And I had ordered it from here https://www.ebay.com.au/itm/133464114562

In hindsight I should have ordered direct from the manufacturer because the ebay listing was notably more expensive than from the manufacturer 😢the links are AU, but surely the same part is available from sellers in other regions

Once I've got the 3d printed case figured out, I'll post it in a new thread on here for anyone to use

**Reply by TalonCard** [reply]
Just a thought. since I have 2x units. I'll try connecting the working screen to the other unit and see where the problem follows! (If I can wrangle the ribbon cable without tearing it.)

**Comment by darianbjohnson** [comment]
Those screen ribbons are notoriously finicky… so go slow. And again; I’m happy to send you another screen to test with!

**Reply by TalonCard** [reply]
Hello again! I had been a bit slack lately, but was able to get around to testing the screen on the faulty unit with the screen on the good unit. The issue followed the LCD module! (I did not expect that, but new LCD tech with it's own quirks I guess)

I'll reach out to you on that link from earlier and we can see about a replacement LCD.
